### PR TITLE
MOB-3819 add shopper locale to adyen payment request

### PR DESCRIPTION
**Pull request by dianakarh**
## JIRA
[MOB-3819](https://karhoo.atlassian.net/browse/MOB-3819)

## DESCRIPTION
[Create a description of what you have changed in this branch]

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [x] Bugfix 
- [ ] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [x] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: KarhooSDK/Api/Request/Adyen/AdyenPaymentMethodsRequest.swift [hunk 1]
```diff
@@ -9,17 +9,23 @@ import Foundation
 
 public struct AdyenPaymentMethodsRequest: Codable, KarhooCodableModel {
     
+    // The platform where a payment transaction takes place. This field can be used for filtering out payment methods that are only available on specific platforms. Possible values: * iOS * Android * Web
```
Comment (AleksanderWedrychowskiKarhoo) [inline KarhooSDK/Api/Request/Adyen/AdyenPaymentMethodsRequest.swift:12]: Shouldn't be this doc added to reference?

Comment (AleksanderWedrychowskiKarhoo) [reply KarhooSDK/Api/Request/Adyen/AdyenPaymentMethodsRequest.swift:12]: Same for shoperLocale

Comment (AleksanderWedrychowskiKarhoo) [reply KarhooSDK/Api/Request/Adyen/AdyenPaymentMethodsRequest.swift:12]: I mean adding one more `/`

**Review by AleksanderWedrychowskiKarhoo — APPROVED** [review]

File: KarhooSDK/Api/Request/Adyen/AdyenPaymentMethodsRequest.swift (hunk 1 above)
Comment (dianakarh) [reply KarhooSDK/Api/Request/Adyen/AdyenPaymentMethodsRequest.swift]: According to the new styleguide there should indeed be 3 `/`. Fixed now in both comments :)

**Review by AleksanderWedrychowskiKarhoo — APPROVED** [review]

**Pull request merged**